draft-ietf-acme-dns-account-01.mkd

```diff
@@ -120,51 +217,55 @@ A client can fulfill this challenge by performing the following steps:
+     - "domain" if the authorization is for both the host and all subdomains by containing the `subdomainAuthAllowed` field set to true ({{?RFC9444, Section 4.1}}).
    - The `"||"` operator indicates concatenation of strings
 
 - Provision a DNS `TXT` record with the base64url digest value under the constructed domain validation name
 
-For example, if the domain name being validated is "www.example.org", and the account URL of "https://example.com/acme/acct/ExampleAccount" then the client would provision the following DNS record:
+  For example, if the domain name being validated is `*.example.org`, and the account URL of `https://example.com/acme/acct/ExampleAccount` then the client would provision the following DNS record:
 
-~~~
-_acme-challenge_ujmmovf2vn55tgye.www.example.org 300 IN TXT "LoqXcYV8...jxAjEuX0.9jg46WB3...fm21mqTI"
-~~~
+  ~~~
+  _ujmmovf2vn55tgye._acme-wildcard-challenge.example.org 300 IN TXT "LoqXcYV8...jxAjEuX0.9jg46WB3...fm21mqTI"
+  ~~~
 
-(In the above, "..." indicates that the token and the JWK thumbprint in the key authorization have been truncated to fit on the page.)
+  (In the above, "..." indicates that the token and the JWK thumbprint in the key authorization have been truncated to fit on the page.)
 
-Respond to the ACME server with an empty object ({}) to acknowledge that the challenge can be validated by the server
+- Respond to the ACME server with an empty object ({}) to acknowledge that the challenge can be validated by the server
 
-~~~
-POST /acme/chall/Rg5dV14Gh1Q
-Host: example.com
-Content-Type: application/jose+json
+  ~~~
+  POST /acme/chall/Rg5dV14Gh1Q
+  Host: example.com
+  Content-Type: application/jose+json
 
-{
-  "protected": base64url({
-    "alg": "ES256",
-    "kid": "https://example.com/acme/acct/evOfKhNU60wg",
-    "nonce": "SS2sSl1PtspvFZ08kNtzKd",
-    "url": "https://example.com/acme/chall/Rg5dV14Gh1Q"
-  }),
-  "payload": base64url({}),
-  "signature": "Q1bURgJoEslbD1c5...3pYdSMLio57mQNN4"
-}
-~~~
+  {
+    "protected": base64url({
+      "alg": "ES256",
+      "kid": "https://example.com/acme/acct/evOfKhNU60wg",
+      "nonce": "SS2sSl1PtspvFZ08kNtzKd",
+      "url": "https://example.com/acme/chall/Rg5dV14Gh1Q"
+    }),
+    "payload": base64url({}),
+    "signature": "Q1bURgJoEslbD1c5...3pYdSMLio57mQNN4"
+  }
+  ~~~
 
 On receiving a response, the server constructs and stores the key authorization from the challenge `token` value and the current client account key.
 
 To validate the `dns-account-01` challenge, the server performs the following steps:
 
 - Compute the SHA-256 digest {{FIPS180-4}} of the stored key authorization
-- Compute the validation domain name with the account URL of the ACME account requesting validation
+- Compute the validation domain name with the account URL of the ACME account requesting validation and the associated authorization similar to the client
```

Comment:
```suggestion
- Compute the validation domain name with the account URL of the ACME account requesting validation and the associated authorization, similar to the client logic
```
I had a bit of difficulty parsing this. I think a comma helps and specifying logic?